### ISSUE TITLE
Reconsider focus flow for interactive repl

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#233924]
@jooyoungseo and I discussed the current behavior when a user:
1. is in a python file, selects text, uses `shift+enter` to execute in the interactive repl
2. is in the interactive repl input and hits `enter` to execute it in the interactive repl

Currently:
- For case 1, focus moves to the executed cell.
- For case 2, focus moves to the executed cell.
- One cannot open the accessible view with the last executed cell's content from within the repl input.
- The output is announced and the audio cue is played.

Proposal:
- For case 1, move focus to the python repl input
- For case 2, **keep** focus in the python repl input
- Allow opening the most recently executed cell in the accessible view when focus is in the repl input
- Add an aria label hint to open the accessible view from within the repl input

This is consistent with how the chat view works with responses.
